modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/display/context/MillerColumnsDisplayContext.java

```diff
@@ -643,6 +643,8 @@ else if (layout.isTypeContent()) {
 					"label",
 					LanguageUtil.get(
 						_httpServletRequest, "export-for-translation")
+				).put(
+					"layoutId", layout.getLayoutId()
```

Comment:
Check this value, it is not correct and the fetch request is failing and preventing the modal to show up.